### PR TITLE
Mds 2614 permit application letters (generate pdf)

### PR DESCRIPTION
**Pull request by Jsyro**
Store template files on backend
Store template_met in db
Serve form spec to front end
Dynamically make form based on that spec. 
Create token to generate file with docgen-api
Redeem token to generate the file and stream the file to CORE

### CODE REVIEW

File: services/core-api/app/api/document_generation/resources/now_document.py [hunk 1]
```diff
@@ -7,56 +7,29 @@
...
-    parser.add_argument('now_application_guid', type=str, location='json', required=True)
-    parser.add_argument('template_data', type=str, location='json', required=True)
-
-    @api.doc(
-        description=
-        'Generates the specified document for the NoW using the provided template data and issues a one-time token that is used to download the document.',
-        params={'document_type_code': 'The code indicating the type of document to generate.'})
-    @api.marshal_with(NOW_DOCUMENT_DOWNLOAD_TOKEN_MODEL, code=200)
-    @requires_role_edit_permit
-    def post(self, document_type_code):
```
Comment (Jsyro) [inline services/core-api/app/api/document_generation/resources/now_document.py]: This is a post on the document-type.

File: services/core-api/uwsgi.ini [hunk 2]
```diff
@@ -1,6 +1,7 @@
 [uwsgi]
 # LOGGING
 route = ^(.*)/health donotlog:
+route = ^(.*)/intake/v2/events donotlog:
```
Comment (Jsyro) [inline services/core-api/uwsgi.ini]: this is the send request that pushed APM, don't dilute logs with this.

File: .vscode/settings.json [hunk 3]
```diff
@@ -10,17 +10,21 @@
...
   },
   "[javascript]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
   },
+  "prettier.trailingComma": "es5",
+  "prettier.arrowParens": "always",
+  "prettier.bracketSpacing": true,
+  "prettier.printWidth": 100,
+  "prettier.endOfLine": "lf",
+
```
Comment (Jsyro) [inline .vscode/settings.json]: remove .prettierjson files and use workspace settings.

File: services/minespace-web/package.json [hunk 4]
```diff
@@ -118,10 +118,10 @@
...
     "lint": "eslint ./src",
     "lint:file": "eslint",
-    "format:check": "prettier --config .prettierrc.json",
-    "format:check-all": "prettier --config .prettierrc.json src/*.js && prettier --config .prettierrc.json src/**/*.js && prettier --config .prettierrc.json src/**/**/*.js",
-    "format:write": "prettier --config .prettierrc.json --write",
-    "format:write-all": "prettier --config .prettierrc.json src/*.js --write && prettier --config .prettierrc.json src/**/*.js --write && prettier --config .prettierrc.json src/**/**/*.js --write",
+    "format:check": "prettier",
+    "format:check-all": "prettier src/*.js && prettier src/**/*.js && prettier src/**/**/*.js",
+    "format:write": "prettier --write",
+    "format:write-all": "prettier src/*.js --write && prettier src/**/*.js --write && prettier src/**/**/*.js --write",
```
Comment (Jsyro) [inline services/minespace-web/package.json]: no more .prettierrc.json file.

**Comment by pdittaro** [comment]
Closing this pull request becuase its creating 38GB of log files in Jenkins

**Comment by Jsyro** [comment]
closing to make fresh one.... frontend docker build is having issues.

**Comment by pdittaro** [comment]
Also, `2000m` is just `2`

**Pull request closed without merging**